e2e/go.mod

```diff
@@ -206,9 +217,15 @@ replace (
 	// Docker Compose
 	// https://github.com/docker/compose/blob/v2.20.2/go.mod#L192
 	github.com/cucumber/godog => github.com/laurazard/godog v0.0.0-20220922095256-4c4b17abdae7
-	// BuildKit
-	// https://github.com/moby/buildkit/blob/v0.12.1/go.mod#L73-L83
-	go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc => go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc v0.41.0
-	go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp => go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp v0.41.0
-	go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracehttp => go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracehttp v1.15.1
+
+	// NOTE(adamtagscherer):
+	// For k8s dependencies, we use a replace directive, to prevent them being
+	// upgraded to the version specified in containerd, which is not relevant to the
+	// version needed.
+	// See https://github.com/docker/buildx/pull/948 for details.
+	// https://github.com/docker/buildx/blob/v0.9.1/go.mod#L62-L64
+	k8s.io/api => k8s.io/api v0.26.2
```

Comment:
Are the k8s replaces needed because we are pulling in `k8s.io/api` in the root `go.mod` and docker-compose is not happy with that version?

Reply:
these were readded as part of a merge conflict, fixed in the latest